INSTALL.md

```diff
@@ -1,15 +1,44 @@
 # PyBitmessage Installation Instructions
 
-For an up-to-date version of these instructions, please visit the
-[Bitmessage Wiki](https://bitmessage.org/wiki/Compiling_instructions).
+## Download
+Download for Windows (32bit) (64bit)
+https://github.com/Bitmessage/PyBitmessage/releases/download/v0.6.1/Bitmessage-0.6.1.exe
 
-PyBitmessage can be run in either one of two ways:
+Download for Apple OS X
+https://github.com/Bitmessage/PyBitmessage/releases/download/v0.6.1/bitmessage-v0.6.1.dmg
 
-- straight from source
+For Linux follow below given steps & for detailed instruction check [Bitmessage Wiki](https://bitmessage.org/wiki/Compiling_instructions).
 
- or
-- from an installed
-package.
+## setuptools
+This is now the recommended and in most cases the easiest procedure for installing PyBitmessage. There is also a helper script to resolve dependencies easily.
+
+Go to the directory with PyBitmessage source code and run:
+```
+python checkdeps.py
+```
+If there are missing dependencies, it will explain you what is missing and for many Unix-like systems also what you have to do to resolve it. How you then run setuptools depends on whether you want to install it to user's directory or system.
+
+As normal user:
+```
+python setup.py install --user
+~/.local/bin/pybitmessage
+```
+or as root:
+```
+python setup.py install
+/usr/local/bin/pybitmessage
+```
+
+## Virtualenv
+Create virtualenv with Python 2.x version
+```
+virtualenv -p python2 env
+```
+
+Activate env
+```
+source env/bin/activate
+```
```

Comment:
incomplete. Needs to add installing dependencies, building pybitmessage, and running it.